editions/tw5.com/tiddlers/messages/WidgetMessage_ tm-modal.tid

```diff
@@ -14,6 +14,15 @@ The "currentTiddler" variable is set to the title of the modal tiddler, but can
 
 The modal message is usually generated with the ButtonWidget. The modal message is handled by the TiddlyWiki core.
 
+! The Modal Tiddler
+Within the Modal tiddler itself;
+
+*The field named "subtitle" will be used as the title of the Modal Window
+**This can contain macros, widgets and transcludions as well
+*The field named "footer" will be used as the footer of the Modal Window
+**The most common footer may be `<$button message="tm-close-tiddler">Close</$button>` Which allows closure of the modal
+*Parameters provided in the action widget such as yourName and yourMessage are accessible within the Modal Tiddler as variable names, for example `<<yourName>>` and `<<yourMessage>>`
```

Comment:
Perhaps clearer to say "The named parameters specified in the tm-modal message are made available as variables within the modal."